Add a local run with minimal config

jacomago opened the pull request.

jacomago commented:
@mdavidsaver What do you think? I was using this to replicate ci locally so I knew exactly what it was doing.

mdavidsaver commented:
Maybe better to restructure the following conditional so that "local" is the `else:` case?

```py
if GITLAB_CI:
    ...
elif TRAVIS:
    ...
else: # implies local
    ...

jacomago committed:
Switch on the ci

Allows a default environment setup
For example when running the scripts locally
